lib/options-manager.js

```diff
@@ -174,6 +175,10 @@ const handleTSConfig = async options => {
 		options.tsConfig = searchResults.config;
 	}
 
+	if (options.tsConfigPath) {
+		options.tsConfigPath = slash(options.tsConfigPath);
```

Comment:
I think the `slash` calls should only be done right before passing some paths to globbing, as it's the globbing that does not support backward slashes, everything else does.

Reply:
In fact it was not required, just needed to update the tests.